src/util/PageRange.cpp

```diff
@@ -3,29 +3,21 @@
 #include <cctype>
 #include <cstdlib>
 
-PageRangeEntry::PageRangeEntry(int first, int last) {
-    this->first = first;
-    this->last = last;
-}
-
-PageRangeEntry::~PageRangeEntry() = default;
-
-auto PageRangeEntry::getLast() const -> int { return this->last; }
-
-auto PageRangeEntry::getFirst() const -> int { return this->first; }
-
 auto PageRange::isSeparator(char c) -> bool { return (c == ',' || c == ';' || c == ':'); }
 
-auto PageRange::parse(const char* str, int pageCount) -> PageRangeVector {
+
+// Todo (rename): what does this? Neither name or code is explaining.
+auto PageRange::parse(std::string_view str, size_t pageCount) -> PageRangeVector {
```

Comment:
I believe this parses the string that's typed in the page range field of the export dialog. See `ExportDialog.cpp`.
![image](https://user-images.githubusercontent.com/1066652/152912480-656ee7b1-4e2a-4c77-a0ba-c03179f0d897.png)

Reply:
Ideally, there should be unit tests for this method.